db/schema_tables.cc

```diff
@@ -9,6 +9,7 @@
 
 #include "db/schema_tables.hh"
 
+#include "seastar/core/future.hh"
```

Comment:
Use \<> for Seastar includes (Seastar is a system library from ScyllaDB's perspective)

Reply:
fixed